### ISSUE TITLE
--file command is not working

### ISSUE THREAD

**Issue by MatterMartyr** [RicterZ/nhentai#345]
--file command seems to be unusable after 0.5.10. I reinstalled nhentai using the method mentioned by ExtremeCaliber in #315 and it didn't solve the problem. Here is my attempt to reproduce it:

Before upgrading:
```
C:\Users\Test\nhentai>nhentai --file=test.txt
[15:18:53] banner: nHentai ver 0.5.9: あなたも変態。 いいね?
[15:18:53] main: Using mirror: https://nhentai.net
[15:18:53] main: Using viewer template "default"
[15:18:54] check_cookie: Login successfully! Your username: xxxxxxxxxxxxx
[15:18:54] doujinshi_parser: Fetching doujinshi information of id xxxxxx
[15:18:55] download: Starting to download doujinshi: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
[15:18:55] start_download: Doujinshi will be saved at ".xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
[15:18:57] download: Starting to download https://i.nhentai.net/galleries/xxxxxxx/1.jpg ...
[15:18:57] download: Starting to download https://i.nhentai.net/galleries/xxxxxxx/2.jpg ...
[15:18:57] download: Starting to download https://i.nhentai.net/galleries/xxxxxxx/3.jpg ...
[15:18:57] download: Starting to download https://i.nhentai.net/galleries/xxxxxxx/4.jpg ...
[15:18:57] download: Starting to download https://i.nhentai.net/galleries/xxxxxxx/5.jpg ...
...
[15:19:28] download: Starting to download https://i.nhentai.net/galleries/xxxxxxx/16.jpg ...
[15:19:28] download: Starting to download https://i.nhentai.net/galleries/xxxxxxx/17.jpg ...
[15:19:29] download: Starting to download https://i.nhentai.net/galleries/xxxxxxx/18.jpg ...
[15:19:29] download: Starting to download https://i.nhentai.net/galleries/xxxxxxx/19.jpg ...
[15:19:32] generate_html: HTML Viewer has been written to "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx\index.html"
[15:19:32] main: All done.
```
After upgrading:
```
C:\Users\Test\nhentai>nhentai --file=test.txt
[15:26:26] banner: nHentai ver 0.5.10: あなたも変態。 いいね?
[15:26:26] main: Using mirror: https://nhentai.net
[15:26:26] main: Using viewer template "default"
[15:26:27] check_cookie: Login successfully! Your username: xxxxxxxxxxxxx
[15:26:27] doujinshi_parser: Fetching doujinshi information of id xxxxxx
[15:26:28] main: Skip download doujinshi because a PDF/CBZ file exists of doujinshi xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
[15:26:28] main: All done.
```
Thanks!

Edit: Everything was in default configuration, no regenerate, no history.

**Comment by MatterMartyr** [comment]
Also, I am currently using 0.5.9 and it works just fine.

**Comment by RicterZ** [comment]
fix on 0.5.11
